src/AppBundle/DataFixtures/ORM/LoadPostFixtures.php

```diff
@@ -99,6 +75,30 @@ private function loadPosts(ObjectManager $manager)
         $manager->flush();
     }
 
+    /**
+     * This method must return an array of fixtures classes
+     * on which the implementing class depends on.
+     *
+     * @return array
+     */
+    public function getDependencies()
+    {
+        return [
+            LoadTagFixtures::class,
+            LoadUserFixtures::class,
+        ];
+    }
+
+    private function addRandomTags(Post $post)
+    {
+        $indexes = (array) array_rand(LoadTagFixtures::$names, mt_rand(1, 3));
```

Comment:
Can we use 0 as the min number of tags ? Tags are optional, so it makes sense to have untagged posts too